README.markdown

```diff
@@ -19,8 +19,16 @@ installed ruby (not system ruby) and a clean gemset dedicated to rspec-dev:
 
 [rbenv](https://github.com/sstephenson/rbenv) is also supported.
 
-Windows users can use [uru](https://bitbucket.org/jonforums/uru).
-
+### Windows Instructions
+Windows users can use [uru](https://bitbucket.org/jonforums/uru) to manage Ruby versions.
+Fork the following projects: 
+	[rspec](https://github.com/rspec/rspec)
+	[rspec-core](https://github.com/rspec/rspec-core)
+	[rspec-expectations](https://github.com/rspec/rspec-expectations)
+	[rspec-mocks](https://github.com/rspec/rspec-mocks)
+	[rspec-rails](https://github.com/rspec/rspec-rails) 
+	[rspec-support](https://github.com/rspec/rspec-support)
```

Comment:
This is not really relevant to Windows, and in any case the point of the `rspec-dev` repo is to take care of this for a developer, (and `rspec-rails` is only necessary to work with Rails, and `rspec` itself isn't necessary at all).